source/collections/FloatVectorList.ooc

```diff
@@ -30,6 +35,19 @@ FloatVector: class extends VectorList<Float> {
 		result count = this count
 		result
 	}
+
+	discreteFourierTransform: static func (input: This) -> This {
```

Comment:
Why was this re-implemented instead of using the [existing implementation](https://github.com/cogneco/ooc-kean/blob/master/source/math/FloatComplex.ooc#L63)?